content/blog/2017/2017-02-14-pipleine-editor-preview.adoc

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "Say 'Hello' to the Blue Ocean Pipeline Editor"
+tags:
+- blueocean
+- editor
+- declarative
+- pipeline
+author: jamesdumay
+---
+
+link:/blog/2016/09/19/blueocean-beta-declarative-pipeline-pipeline-editor/[Back in September 2016 we announced the availability of the Blue Ocean beta]
+and the forthcoming Pipeline Editor. We are happy to announce that you can try
+the Pipeline Editor preview today.
+
+=== What is it?
+
+The Pipeline Editor is the simplest way for anyone wanting to get started with
+creating Pipelines in Jenkins. It's also great way for advanced Jenkins users
+to start adopting Pipeline. It allows developers to break up their Pipeline into different
+ stages and parallelize tasks that can occur at the same time - graphically.
+ The rest is up to you.
+
+Thats right, the days of writing Pipelines in your text editor are over. You can do it
```

Comment:
Not sure about this - reads like a use car sales pitch which could work if there was some irony. Perhaps a "but wait, there's more!" (which could be funny actually). Also the "from blue ocean" feels bad, but not sure why. Just creates a division from Jenkins I don't think it is appropriate. 

I would reword this to be:

"A pipeline you create or edit visually can produce a link://blog/2017/02/03/declarative-pipeline-ga/[Declarative Pipeline]`Jenkinsfile` for you. This is a "round trip" editor which lets you load the pipeline back in later, for more editing. Not sure what a Jenkins Pipeline is? Why not checkout our link:/doc/[new guided tour]!"